### PR TITLE
feat: add region migration doc

### PR DESCRIPTION
**Pull request by WenyXu**
## What's Changed in this PR

*Describe the change in this PR*

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [x] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.6/zh/user-guide/operations/region-migration.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Region migration
+
+Region migration allows users to move regions between the Datanode.
+
+:::warning Warning
+This feature is only available on greptimedb running on cluster mode and
```
Comment (nicecui) [inline docs/v0.6/zh/user-guide/operations/region-migration.md:6]:
```suggestion
This feature is only available on GreptimeDB running on cluster mode and 
```

File: docs/v0.6/zh/user-guide/operations/region-migration.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Region migration
+
+Region migration allows users to move regions between the Datanode.
+
+:::warning Warning
+This feature is only available on greptimedb running on cluster mode and 
+- Using Kafka WAL
+- Using shared storage (e.g., AWS S3)
```
Comment (nicecui) [inline docs/v0.6/zh/user-guide/operations/region-migration.md:8]:
```suggestion
- Using [shared storage](/user-guide/operations/configuration.md#storage-options) (e.g., AWS S3)
```

File: docs/v0.6/en/user-guide/operations/region-migration.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| region id     | leader_peer                                                  | leader_peer_id |
+|---------------|--------------------------------------------------------------|----------------|
+| 4668629450752 | greptimedb-datanode-2.greptimedb-datanode.my-greptimedb:4001 | 2              |
+
+## 选择 Region 迁移的目标节点
+:::warning Warning
+当起始节点等于目标节点时，Region 迁移不会被执行
+:::
+
+如果你通过 GreptimeDB operator 部署 DB 集群，Datanode 的 `peer_id` 总是从 0 开始递增。例如，有一个3 Datanode DB 集群，则 `peer_id` 应为 0,1,2。
```
Comment (nicecui) [inline docs/v0.6/en/user-guide/operations/region-migration.md:38]:
```suggestion
如果你通过 GreptimeDB operator 部署 DB 集群，Datanode 的 `peer_id` 总是从 0 开始递增。例如，DB 集群有 3 个 Datanode，则 `peer_id` 应为 0,1,2。
```

**Review by nicecui — COMMENTED** [review]
The location of the Chinese and English documents is reversed 😂

**Comment by WenyXu** [comment]
> The location of the Chinese and English documents is reversed 😂

Hhh, interesting. 🤣

**Comment by WenyXu** [comment]
@nicecui PTAL

**Review by nicecui — APPROVED** [review]

**Review by Wenjie0329 — APPROVED** [review]

**Pull request merged**